erpnext/selling/doctype/selling_settings/selling_settings.json

```diff
@@ -200,14 +194,20 @@
    "fieldname": "allow_negative_rates_for_items",
    "fieldtype": "Check",
    "label": "Allow Negative rates for Items"
+  },
+  {
+   "description": "Percentage you are allowed to <b>sell</b> beyond the Blanket Order quantity.",
```

Comment:
Same here. We’re on the Selling Settings page. There may not be any confusion of the two, as the context is there.